### DISCUSSION TITLE
Excited for 1.0!

### DISCUSSION THREAD

**Discussion by jess-sol** [zesterer/chumsky#516] [Show and tell]
I've been working with Chumsky for a few weeks now writing a parser for a Yaml like language. I had to switch to main to get the context related stuff to parse indentation correctly. I just wanted to say it's been a joy so far working with Chumsky! I just started needing `map_ctx`, and thought I'd have to implement it myself, but actually I just had to do a `cargo update` :)

Really excited for the 1.0 release, and I just wanted to say I appreciate the effort being put into it

**Comment by zesterer** [comment]
Thanks for your kind words, I'm happy the crate is working out well for you! I'm also excited for 1.0. There's still a list of things to prepare for release, but what's currently on `main` and the alpha releases is unlikely to change much so you can be reasonably confident that working on top of it shouldn't result in dramatic breakage come the final release :)
